### PR TITLE
Add invertbuttons option (Confirm / Prompt)

### PR DESCRIPTION
**Pull request by danicomas**
- defaults: Invert buttons for all confirms / prompts
- specific: Invert buttons for specific confirm / prompt

### CODE REVIEW

**Comment by danicomas** [comment]
Now It is OK @makeusabrew . Ready to be merged!

**Comment by danicomas** [comment]
Now It is OK @makeusabrew . Ready to be merged!

**Comment by Austin-Qu** [comment]
Anyone still remember this? It's a quite helpful option indeed. @makeusabrew @danicomas

**Comment by danicomas** [comment]
I am waiting to be merged @Austin-Qu. It depends on @makeusabrew.

**Comment by danicomas** [comment]
@makeusabrew can be merged?

**Comment by makeusabrew** [comment]
Hi @danicomas,

Sorry for not getting back to you. I know I've delayed this massively; something about it just doesn't feel quite right. I don't like manipulating the DOM after creating the dialog too - why can't we just swap the order the buttons are created in? Doing it 'retrospectively' *after* the DOM has been created feels like a hack an external library would have to use, not one the library itself should need.

Would appreciate some perspective - @tarlepp what do you think?

File: bootbox.js [hunk 1]
```diff
@@ -317,9 +319,15 @@
 
   exports.confirm = function() {
     var options;
+    var invertButtons = defaults.invertButtons;
+    var dialog;
 
     options = mergeDialogOptions("confirm", ["cancel", "confirm"], ["message", "callback"], arguments);
```
Comment (makeusabrew) [inline bootbox.js]: RE my main comment about swapping the button order, we could easily just swap the array passed as the second param here...

Comment (danicomas) [inline bootbox.js]: But the cancel and confirm button would have the active button inverted isn't it? @makeusabrew

Comment (makeusabrew) [inline bootbox.js]: Fair point :). I haven't had proper dig back through my own code in a while, sorry. Looking at it again, sanitize(options) is always called and it's this which decides button classes; so this method would know about options.invertButtons too. Maybe I'm overreacting to the DOM 'hack' - interested in getting a bit more feedback first (i.e. from @tarlepp) - sorry to keep you waiting even longer.

Comment (danicomas) [inline bootbox.js]: OK @makeusabrew . We will wait for @tarlepp feedback :D

**Comment by tarlepp** [comment]
Hmm, I'm going take some serious time with this now. Need some investigation and reading...

**Comment by tarlepp** [comment]
I don't like that DOM manipulation on those buttons. Imho it would be just nicer to swap that button array, and then refactor default button class handling so that those won't be done like this: https://github.com/makeusabrew/bootbox/blob/master/bootbox.js#L175-L182

**Comment by makeusabrew** [comment]
Declaring PR bankruptcy here. This issue is just too old to deal with now. Please do open a new issue or PR if it's still relevant :). Thanks!

**Pull request closed without merging**